looker/resource_looker_group_user_test.go

```diff
@@ -0,0 +1,129 @@
+package looker
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	client "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+	sdk "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+	"github.com/resolutionlife/terraform-provider-looker/internal/conv"
+	"github.com/resolutionlife/terraform-provider-looker/internal/slice"
+)
+
+func init() {
+	// Add a sweeper to remove groups and users that have names starting with `test-acc`.
```

Comment:
Would it be sufficient to delete the groups and users that start with `test-acc`? That way the binding between the two has been deleted.

Reply:
I suppose it would, but I was just trying to be thorough. I'll give a try without doing that as well.